introduction.md

```diff
@@ -0,0 +1,2 @@
+hey its me and i'm making this file because i have to, to complete this course
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.